configs/components/jsbach/jsbach.datasets.yaml

```diff
@@ -60,38 +60,32 @@ choose_dataset:
-                                "${input_dir_awi}/${echam.resolution}/Ndepo/ssp245/${echam.resolution}_ndepo.input4MIPs_surfaceFluxes_ScenarioMIP-NCAR-CCMI-ssp245_2099.nc": # 12 months
-                                        from: 2100
-                                        to: 2101
-                        "Ndepo_ssp534os":
-                                #"${input_dir}/${echam.resolution}/${echam.resolution}_ndepo.input4MIPs_surfaceFluxes_ScenarioMIP-NCAR-CCMI-ssp534os.nc": # nyear x nmonth
-                                "${input_dir_awi}/${echam.resolution}/Ndepo/ssp534os/${echam.resolution}_ndepo.input4MIPs_surfaceFluxes_ScenarioMIP-NCAR-CCMI-ssp534os_@YEAR@.nc": # 12 months; see README.md in Ndepo/ dir
-                                        from: 1850
-                                        to: 2099
-                                "${input_dir_awi}/${echam.resolution}/Ndepo/ssp534os/${echam.resolution}_ndepo.input4MIPs_surfaceFluxes_ScenarioMIP-NCAR-CCMI-ssp534os_2099.nc": # 12 months
-                                        from: 2100
-                                        to: 2101
-                        "Ndepo_ssp585":
-                                #"${input_dir}/${echam.resolution}/${echam.resolution}_ndepo.input4MIPs_surfaceFluxes_ScenarioMIP-NCAR-CCMI-ssp585.nc": # nyear x nmonth
-                                "${input_dir_awi}/${echam.resolution}/Ndepo/ssp585/${echam.resolution}_ndepo.input4MIPs_surfaceFluxes_ScenarioMIP-NCAR-CCMI-ssp585_@YEAR@.nc": # 12 months; see README.md in Ndepo/ dir
+
+                        "Ndepo_hist_scenario":
+                            # TODO: pool on Levante seems to be broken currently.
+                            # There are no yearly Ndepo files. ${input_dir_awi}
+                            # contains the yearly files
+                            choose_use_input_dir_awi:
+                                False:
+                                    "${input_dir}/${echam.resolution}/ndepo/Ndepo.1850.nc":
+                                        to: 1850
+                                    "${input_dir}/${echam.resolution}/ndepo/Ndepo.@YEAR@.nc":
+                                        from: 1851
+                                        to: 2014
+                                    # TODO: these files don't exist on Levante
+                                    "${input_dir}/${echam.resolution}/ndepo/Ndepo_${echam.scenario}.@YEAR@.nc":
+                                        from: 2015
+                            # TODO: test this with a check run
+                                True:
+                                    # These files are 12-monthly; see README.md in Ndepo/ directory
+                                    # NOTE: ssp370 might be missing in this directory
+                                    "${input_dir_awi}/${echam.resolution}/Ndepo/${echam.scenario}/${echam.resolution}_ndepo.input4MIPs_surfaceFluxes_ScenarioMIP-NCAR-CCMI-${echam.scenario}_@YEAR@.nc": 
```

Comment:
This choose won't be reached and it won't be resolved. Choose cannot be indented anywhere, except for sections (models, computer, general, ...) or other chooses.

I suggest the following:
1. remove the `choose`
2. leave `Ndepo_hist_scenario` as it was
3. create a new `Ndepo_ssp` label and add the changes there